hack/make-devenv.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+
+# Copyright 2022 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+IMAGE_TAG="melange-inception"
+DEVENV_IMAGE_TARBALL="melange-inception.tar.gz"
+BUILDER_APKO_TAG="sha256:c96d1e6886ae5bafafe2656a3133ee73ce5245f67e60e430a731570b3f76797e"
```

Comment:
wondering if latest is more appropriate, since this will get out of date in < 24 hrs

Reply:
Updated to latest